### PR TITLE
Add Standard Library changes to the release note of Swan Lake Beta 1

### PR DESCRIPTION
**Pull request by Maninda**
$subject.

### CODE REVIEW

File: release-notes/swan-lake-betarc1-release-note.md [hunk 1]
```diff
@@ -55,12 +55,29 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Beta RC1](https://git
 
 ### Standard Library Updates
 
-#### New Features
-
 #### Improvements
 
+##### GraphQL Package
```
Comment (praneesha) [inline release-notes/swan-lake-betarc1-release-note.md:60]:
```suggestion
##### `graphql` Package
```

File: release-notes/swan-lake-betarc1-release-note.md [hunk 2]
```diff
@@ -55,12 +55,29 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Beta RC1](https://git
...
+
+ - Allow enum as an input parameter.
+
 #### Bug Fixes
 
+##### GraphQL Package
```
Comment (praneesha) [inline release-notes/swan-lake-betarc1-release-note.md:66]:
```suggestion
##### GraphQL Package
```
We do not mention bug fixes specifically in the current format.

File: release-notes/swan-lake-betarc1-release-note.md [hunk 3]
```diff
@@ -55,12 +55,29 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Beta RC1](https://git
...
+
+ - Fix service crashing issue when types have recursive reference.
+ - Fix service hanging issue when a GraphQL field is of a union type including service type.
+ - Fix service hanging issue when a GraphQL field returns a service array.
+
+##### WebSocket Package
```
Comment (praneesha) [inline release-notes/swan-lake-betarc1-release-note.md:72]:
```suggestion
##### WebSocket Package
```
We do not mention bug fixes specifically in the current format.

File: release-notes/swan-lake-betarc1-release-note.md [hunk 4]
```diff
@@ -55,12 +55,29 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Beta RC1](https://git
...
+
+ - Fix the issue of WebSocket services not working when Observability is enabled.
+ - Fix the listener initialization failure when configurations are given as included record params.
+
+##### Email Package
```
Comment (praneesha) [inline release-notes/swan-lake-betarc1-release-note.md:77]:
```suggestion
##### Email Package
```
We do not mention bug fixes specifically in the current format.

File: release-notes/swan-lake-betarc1-release-note.md (hunk 2 above)
Comment (Maninda) [reply release-notes/swan-lake-betarc1-release-note.md]: Removed.

File: release-notes/swan-lake-betarc1-release-note.md (hunk 3 above)
Comment (Maninda) [reply release-notes/swan-lake-betarc1-release-note.md]: Removed.

File: release-notes/swan-lake-betarc1-release-note.md (hunk 4 above)
Comment (Maninda) [reply release-notes/swan-lake-betarc1-release-note.md]: Removed.

**Review by praneesha — APPROVED** [review]

**Pull request merged**